cognite/client/_api/transformations/schema.py

```diff
@@ -34,17 +39,76 @@ def retrieve(
                 >>> from cognite.client.data_classes import TransformationDestination
                 >>> c = CogniteClient()
                 >>> columns = c.transformations.schema.retrieve(destination = TransformationDestination.assets())
+
+            Get the schema for a transformation producing nodes::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import TransformationDestination
+                >>> from cognite.client.data_classes.transformations.common import ViewInfo
+                >>> c = CogniteClient()
+                >>> view = ViewInfo(space="viewSpace", external_id="viewExternalId", version="viewVersion")
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.nodes(view, "InstanceSpace")
+
+            Get the schema for a transformation producing edges::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import TransformationDestination
+                >>> from cognite.client.data_classes.transformations.common import ViewInfo
+                >>> c = CogniteClient()
+                >>> view = ViewInfo(space="viewSpace", external_id="viewExternalId", version="viewVersion")
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.edges(view, "InstanceSpace")
+
+            Get the schema for a transformation producing instance type data model::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import TransformationDestination
+                >>> from cognite.client.data_classes.transformations.common import DataModelInfo
+                >>> c = CogniteClient()
+                >>> data_model = DataModelInfo(space="dataModelSpace", external_id="dataModelExternalId",version="dataModelVersion",destination_type="type")
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.instances(data_model, "InstanceSpace")
+
+            Get the schema for a transformation producing instance relationship data model::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import TransformationDestination
+                >>> from cognite.client.data_classes.transformations.common import DataModelInfo
+                >>> c = CogniteClient()
+                >>> data_model = DataModelInfo(space="dataModelSpace", external_id="dataModelExternalId",version="dataModelVersion",destination_type="type", destination_relationship_from_type="relationshipFromType")
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.instances(data_model, "InstanceSpace")
```

Comment:
Might have an example using the new parameters?

Reply:
In the user document https://cognite-sdk-python.readthedocs-hosted.com/en/latest/transformations.html#create-transformations, there are no examples with real data for the Transformation `create`. 

If I added real data examples, then the format will break. Please share you opinion.